src/screens/Interests.tsx

```diff
@@ -1,13 +1,19 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import { Button, Chip } from "@patternfly/react-core";
 import { useNavigate } from "react-router-dom";
 // Import firebase
 import { initializeApp } from 'firebase/app';
 import { getFirestore } from 'firebase/firestore';
 import { config } from '../config/config';
 import { doc, getDoc,setDoc } from "firebase/firestore";
-import linksJson from "../links.json";
 import { ProgressStepperCompact4 } from "../components/home/Progressbar";
+import { APIUrl } from "./Home";
+
+type interest = {
+  title: string;
+  selected: boolean;
+};
+
```

Comment:
for style reasons, lets use interfaces. It also means that it should be re-named to `IInterest` since we named anything that's an interface in CamcelCase with a leading uppercase `I` to indicate that it's an interface(not a type of class)